test/mocks/director.js

```diff
@@ -452,4 +453,27 @@ function getDeploymentVms(deploymentName, times, vms, boshDirectorUrlInput, foun
         index: 0
       }]);
   }
+}
+
+function getDeploymentInstances(deploymentName, times, vms, boshDirectorUrlInput, found) {
+  const boshDirectorUrl = boshDirectorUrlInput || directorUrl;
+  if (found === false) {
+    return nock(boshDirectorUrl)
+      .get(`/deployments/${deploymentName}/instances`)
+      .times(times || 1)
+      .reply(404, {
+        'code': 70000,
+        'description': `'Deployment ${deploymentName} doesn\'t exist'`
+      });
+  } else {
+    return nock(boshDirectorUrl)
+      .get(`/deployments/${deploymentName}/instances`)
+      .times(times || 1)
+      .reply(200, vms || [{
+        cid: '081e3263-e066-4a5a-868f-b420c72a260d',
```

Comment:
does getDeploymentInstances() response actually return cid? If not, we can remove it from the mock to avoid confusion.

Reply:
the response does contain cid - https://bosh.io/docs/director-api-v1.html#list-instances. The response of /vms is the same as /instances except that cid & agent id would be null in case the vm is deleted.

Reply:
ok.. got it..